cmd/podman/shared/funcs.go

```diff
@@ -24,10 +24,14 @@ func GenerateCommand(command, imageName, name string) []string {
 			newArg = imageName
 		case "IMAGE=IMAGE":
 			newArg = fmt.Sprintf("IMAGE=%s", imageName)
+		case "IMAGE=$IMAGE":
+			newArg = fmt.Sprintf("IMAGE=%s", imageName)
 		case "NAME":
 			newArg = name
 		case "NAME=NAME":
 			newArg = fmt.Sprintf("NAME=%s", name)
+		case "NAME=$NAME":
+			newArg = fmt.Sprintf("NAME=%s", name)
```

Comment:
If we want to treat it like this we should also cover separated "$IMAGE" here. As the atomic cli support it here is the example label:
```
"install": "/usr/bin/docker run --rm $OPT1 --privileged -v /:/host -e HOST=/host -e NAME=$NAME -e IMAGE=$IMAGE $IMAGE $OPT2 /usr/bin/install.sh  $OPT3",
```

Reply:
i believe that gets taking care of with env substitutions

Reply:
Hi @baude 
Seems this is not covered bu env. We need fix this:
```
# podman container runlabel -p --tls-verify=false install brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7/etcd:latest
Command: /proc/self/exe run --rm  --privileged -v /:/host -e HOST=/host -e NAME=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7/etcd:latest -e IMAGE=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7/etcd:latest   /usr/bin/install.sh  
unable to pull : error getting default registries to try: repository name must have at least one component
`/proc/self/exe run --rm  --privileged -v /:/host -e HOST=/host -e NAME=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7/etcd:latest -e IMAGE=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/rhel7/etcd:latest   /usr/bin/install.sh  ` failed: exit status 125

```